contentful/client.py

```diff
@@ -219,10 +219,14 @@ def entry(self, entry_id, query=None):
 
         try:
             query.update({'sys.id': entry_id})
-            return self._get(
+            response = self._get(
                 self.environment_url('/entries'),
                 query
-            )[0]
+            )
+            if self.raw_mode:
+                return response
+            else:
```

Comment:
No need for this else statement, just do `return response[0]`.

Also please add a blank line between the end of the function call and the returns section.